src/components/Q&A/QuestionComp.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import AddAnswer from './AddAnswer.jsx';
+import {API_KEY} from '../../config/config.js';
+const axios = require('axios');
+
+class QuestionComp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      questions: this.props.questionList,
+      helpClick: false,
+      showAns: false
+    }
+    this.handleQHelp = this.handleQHelp.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+  }
+
+
+  handleQHelp(qObj) {
+    var temp = parseInt(qObj.question_id);
+    console.log(temp);
+    axios.put(`https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfc/qa/questions/${temp}/helpful`,
```

Comment:
I'd recommend using opt + shift + F to automatically fix the indentation on this axios request to improve readibility.

Reply:
Didn't even know about that shortcut, thanks for the tip!